src/components/App.vue

```diff
@@ -44,36 +50,37 @@ export default {
   }
 
   $edge-board-size: 100px;
+  $admin-board-size: constants.$reset-button-size;
```

Comment:
I used the word "size" when naming `reset-button-size` because it's a square and the width/height are the same. IMO `admin-board-width` would be more appropriate here

As an afterthought should we rename `edge-board-size`? Since we have  anew edge board that is not the same size as the others?

Reply:
welp I agree, but I can't come up with a better name than `edge-board-size`, so it's sticking around unless you have any suggestions?

I also considered renaming `AdminBoard` since it maybe doesn't feel like a whole board, but I've got nothing better for that either. I think I need more coffee ☕ ☕ ☕

Reply:
I think AdminBoard is fine until it gets another use.